testing/autoreleasepool_test.cc

```diff
@@ -0,0 +1,7 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/testing/autoreleasepool_test.h"
+
+namespace flutter::testing {}  // namespace flutter::testing
```

Comment:
Just curious, what's this file for?

Reply:
Apparently, to test your reviewing reflexes :) Originally I was going to implement what it was I needed to implement in there, but after banging up the header, I realised there was nothing I needed to implement in there, but then neglected to delete it! Thanks for spotting!